scripts/manila-consistency.py

```diff
@@ -234,5 +259,18 @@ def main():
         log.info("- share group type share type mapping is consistent")
 
 
+    wrong_share_instance_access_mapping = get_wrong_share_instance_access_mapping(manila_metadata)
+    if len(wrong_share_instance_access_mapping) != 0:
+        log.info("- share instance access mapping inconsistencies found")
+        # print out what we would delete
+        for share_instance_access_mapping_id in wrong_share_instance_access_mapping:
```

Comment:
you could also access both key and vaule from the `wrong_share_instance_access_mapping.items()`, since the value is used later